packages/core/src/index.ts

```diff
@@ -1,9 +1,13 @@
 export {
-  Injectable,
-  InjectableValue,
-  InjectableDependencies,
+  type Injectable,
+  type InjectableValue,
+  type InjectableDependencies,
+  type InjectableWithName,
+  type InjectableWithoutName,
+  type InjectableValueWithoutName,
```

Comment:
I think that must be `InjectableDependencyWithoutName`, because it describes an entry of an `DependencyTree`.

Reply:
Changed to `DependencyWithoutName` to avoid very long names in result d.ts file